src/github/index.ts

```diff
@@ -67,7 +68,7 @@ interface Paginate extends Octokit.Paginate {
   ): Promise<any[]>
 }
 
-type Graphql = (query: string, variables?: Variables, headers?: Headers) => Promise<GraphQlQueryResponse>
+type Graphql = (query: string, variables?: Variables, headers?: Headers) => ReturnType<ReturnType<typeof withCustomRequest>>
```

Comment:
would it maybe be more clear if we required both `{ withCustomRequest, graphql }` and then used `ReturnType<graphql>` here instead of `ReturnType<ReturnType<typeof withCustomRequest>>`?

Reply:
Fixed in a6a59f7b1e86e8397bbd4e6bbf8250bcec401f29.